arch/arm/src/sama5/sam_tsd.c

```diff
@@ -630,7 +680,7 @@ static void sam_tsd_bottomhalf(void *arg)
       pressr = sam_adc_getreg(priv->adc, SAM_ADC_PRESSR);
 #endif
       /* Discard any bad readings.  This check may not be necessary. */
-
+#if 1
```

Comment:
Why to keep this "#if 1" ? Was it a test that succeed and you forgot to remove of #if 1?

Reply:
I forgot, yes - I always search for the #if 0 I use when I'm suspicious of a bit of code being a problem, but will add a search for #if 1 in future!

Reply:
done